pandas/tests/indexes/test_interval.py

```diff
@@ -491,6 +649,207 @@ def testcontains(self):
         assert not i.contains(20)
         assert not i.contains(-20)
 
+        # Why are there two tests called test contains? this should be cleared up a little, no?
+
+
+    @pytest.mark.xfail(reason="new indexing tests for issue 16316")
+    def test_interval_covers_interval_updated_behavior(self):
+
+        # class Interval:
+        #     def covers(self, other: Interval) -> bool
+
+        assert     Interval(1, 3).covers(Interval(1.5, 2.5))
```

Comment:
can you format those according to PEP8 a well? (it will give a bit less nice alignment for all test cases, but Travis will fail on this)